build/__stagingRuntimeConstants.sh

```diff
@@ -1,6 +1,6 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
-DOTNETCORE_STAGING_RUNTIME_VERSIONS=("8.0")
+DOTNETCORE_STAGING_RUNTIME_VERSIONS=()
```

Comment:
can we update this https://github.com/microsoft/Oryx/blob/main/build/constants.yaml#L482 constant and then re-generate the constants? I think that should actually update it throughout the codebase so it won't get staged anymore

Reply:
Ah good catch -- I'm running the pipeline to create new base runtime images for .NET that are on the public MCR instance, so I'll re-generate the constants as a part of this and get your comment resolved with it. Thanks!